f3df/src/renderables.rs

```diff
@@ -87,6 +87,7 @@ pub fn convert_sector(sector: &crate::Sector) -> Sector {
             let i = cell_index_i_j % (cell_size[0]);
             let j = cell_index_i_j / (cell_size[0]);
             let k = cell_index / (cell_size_i_j);
+            // TODO this offset might be wrong
```

Comment:
Can you add some context to why you think so?

Reply:
Done